services/workflows-service/src/events/workflow-completed-webhook-caller.ts

```diff
@@ -49,18 +49,20 @@ export class WorkflowCompletedWebhookCaller {
       id: data.runtimeData.id,
     });
 
-    const webhooks = getWebhooks(
-      data.runtimeData.config,
-      this.configService.get('ENVIRONMENT_NAME'),
-      'workflow.completed',
-    );
-
     const customer = await this.customerService.getByProjectId(data.runtimeData.projectId, {
       select: {
         authenticationConfiguration: true,
+        subscriptions: true,
       },
     });
 
+    const webhooks = getWebhooks(
```

Comment:
Can we change the `getWebhooks` function to receive names parameters?
Something like:
```suggestion
    const webhooks = getWebhooks({
      config: ...,
      subscriptions: ...
      envName: ...
      eventName: ...
    });
```